src/main/resources/application.properties

```diff
@@ -1,6 +1,12 @@
-quarkus.resteasy.path=/api
 quarkus.log.file.enable=false
 quarkus.http.cors=true
 io.quarkus.code.quarkus-version=${version.quarkus}
 io.quarkus.code.quarkus-platform-version=${version.quarkus-platform}
-io.quarkus.code.git-commit-id=${git.commit.id.abbrev}
\ No newline at end of file
+io.quarkus.code.git-commit-id=${git.commit.id.abbrev}
+
+quarkus.dynamodb.endpoint-override=http://localhost:8000
+
+quarkus.dynamodb.aws.region=eu-central-1
```

Comment:
Could you comment this?

It would be great to have a feature activation only when the properties is set (like for the github).

And as always your favorite part:
- Could you add some tests for the endpoint (with mock)
- Do you have any idea on how we could have tests for the storage part?